mirrord/agent/iptables/src/mesh/exclusion.rs

```diff
@@ -108,16 +105,19 @@ where
     async fn mount_entrypoint(&self) -> IPTablesResult<()> {
         // Should mount the internal iptables because exclusion inserts itself as first rule and
         // this may also happen here so exclusion should mount second
-        self.inner.mount_entrypoint().await?;
+        let first_res = self.inner.mount_entrypoint().await;
```

Comment:
Do we want to mount the second if we failed the first? I agree for doing unmounts on fail but for mounting it's probably unnecessary